src/App.tsx

```diff
@@ -93,11 +107,13 @@ function App(): JSX.Element {
         {todoList.length > 0 ? (
           todoList.map((item) => (
             <TodoList
-              deleteItemFromList={() => deleteItemFromList(item.id)}
+              deleteItemFromList={deleteItemFromList}
               updateItemFromList={updateItemFromList}
               id={item.id}
               item={item.name}
               key={item.id}
+              checked={item.checked}
```

Comment:
As discussed earlier, we are not supposed to send this individually. We have an object which is `{id, name, checked}`. Why are sending them individually instead of sending the entire object.